Assets/Mirror/Runtime/Compression.cs

```diff
@@ -0,0 +1,247 @@
+using UnityEngine;
+
+namespace Mirror
+{
+    /// <summary>
```

Comment:
why do the top level comments use xml?
aren't they just for our understanding?

Reply:
docfx uses xml. If we include comments explaining the class as part of the documentation then it will show up in google search when people are looking for stuff.

It also helps to explain what the code does so that users don't have to work with a black box